homeassistant/components/brother/const.py

```diff
@@ -71,7 +75,7 @@
     ),
 }
 
-SENSOR_TYPES: dict[str, SensorDescription] = {
+SENSOR_TYPES: Final[dict[str, SensorDescription]] = {
```

Comment:
Why was the `Final` not required before?

Reply:
Honestly, I didn't know I should use it.

Reply:
Why should we use it then?

Reply:
Is it to mark the dict as a constant?

Reply:
Yes https://github.com/home-assistant/core/pull/49898#discussion_r623878974